lib/team_stats.rb

```diff
@@ -27,4 +27,20 @@ def create_teams_hash
   def find_by_id(id)
     @teams_hash[id]
   end
+
+  def count
+    @teams.count
```

Comment:
I think using `size` or `length` is faster since `count` is an enumerables so it traverses the array to count how long it is vs just returning it.